docs/data.rst

```diff
@@ -0,0 +1,146 @@
+For example, you must set ``timescale`` to ``hourly``, not ``Hourly``.
+
+Lastly, you'll need to retrive the data: :: 
+
+   data = app.retrieve()
+
+
+Use a csv config file
+**********************
+The :py:func:`climakitae.Application.retrieve_from_csv()` method can be used to retrieve data from 
+a csv configuration file. It just takes the filepath to the csv as an argument. Depending on the number of 
+rows in the csv, different data types can be returned. If the csv has one row, a single :py:class:`xarray.DataArray`
+object will be returned. If the csv has multiple rows, we assume you want to retrieve **multiple** datasets. 
+Set the function argument ``merge`` to ``False`` to return a list of :py:class:`xarray.DataArray` objects, or 
+merge to ``True`` (the default value) to return a single :py:class:`xarray.Dataset` object.
+
+The csv file needs to be configured in a particular way in order for the function to properly read it in. 
+The row values must match valid options in our data catalog, and the headers of the csv must be labelled 
+**exactly** as they are in the following example: 
+
+.. list-table::
+   :widths: 5 5 5 5 5 5 5 5 5 5 
+   :header-rows: 1
+
+   * - variable
+     - units
+     - scenario_historical
+     - scenario_ssp
+     - area_average
+     - timescale 
+     - resolution
+     - time_slice
+     - area_subset
+     - cached_area
+   * - Air Temperature at 2m
+     - degF
+     - Historical Climate
+     - SSP 3-7.0 -- Business as Usual
+     - Yes
+     - hourly
+     - 9 km
+     - (2005, 2025)
+     - states 
+     - CA
+
+Read the data into memory 
+###########################
+The data is retrieved as lazily loaded Dask arrays until you chose to read the data into 
+memory. You'll want to read your data into memory before plotting it, exporting it,
+or performing certain computations in order to optimize performance. To read the data 
+into memory, use the :py:func:`climakitae.Application.load()` method. ::
+
+   data = app.retrieve() 
+   data = app.load(data)
+
+
+Create a quick visualization of the data 
+#########################################
+Once you've retrieved the data and read it into memory, you can generate a quick visualization 
+of the data using the :py:func:`climakitae.Application.view()` method. An appropriate visualization
+will be automatically generated depending on the dimensionality of the input data. ::
+
+   app.view(data)
+
+You can also set the colormap and size of the output visualization using the function arguments; see 
+the documentation in the API for more information. 
+
+Export the data 
+################
+To export the data, first chose the filetype you want to export the data to using the 
+:py:func:`climakitae.Application.export_as()` dropdown menu. This will allow you to choose 
+between three options: netcdf, csv, and geotiff.::
+
+   app.export_as() 
+
+After selecting your desired output filetype, input the data you want to export and the 
```

Comment:
To be thorough here, it would be nice to repeat some of the markdown Beth wrote describing what data structures are suited to each export type (at a minimum that netcdf is most flexible, but csv is very restricted).

Reply:
good idea!